index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 My Portfolio</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body class="bg">
+  <div class="wrapper">
+    <div class="toolbar">
+      <div class="mylogo">My Logo</div>
+      <div class='hamburger'>
+        <div class='line'></div>
+        <div class='line'></div>
+        <div class='line'></div>
+      </div>
+    </div>
+     	<p class="myheader">I’m PRATAP Glad to see you!</p>
+    	<p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    	<p class="connect">LET’S CONNECT</p>
+    <div class="social-media">
+      <p class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='36' height="36" alt='' title=''></a></p>
+      <p class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='36' height="36" alt='' title=''></a></p>
+      <p class="sm"><a href=""><img src='images/m.svg'  width='36' height="36" alt='' title=''></a></p>
+      <p class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='36' height="36" alt='' title=''></a></p>
+      <p class="sm"><a href=""><img src='images/v.svg'  width='36' height="36" alt='' title=''></a></p>
+    </div>
```

Comment:
- [ ] The container of the headline section (right after the header) should be a `<section>` tag.